pkg/image/qemu.go

```diff
@@ -136,7 +136,16 @@ func (o *qemuOperations) Resize(image string, size resource.Quantity) error {
 }
 
 func (o *qemuOperations) Info(image string) (*ImgInfo, error) {
-	output, err := qemuExecFunction(qemuInfoLimits, nil, "qemu-img", "info", "--output=json", image)
```

Comment:
Was the rate limited http server actually causing this to time out?  If so, cool, I guess.

Reply:
Yeah, the default timeout of 5 seconds was not enough with the rate limited http server to get all the information qemu-img info wanted to obtain. I was surprised as well.